files/downloads/index.html

```diff
@@ -5,127 +5,501 @@
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.16.1/index.html'>1.7.16.1</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.16.2/index.html'>1.7.16.2</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.16/index.html'>1.7.16</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.17/index.html'>1.7.17</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.18/index.html'>1.7.18</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.19/index.html'>1.7.19</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.20/index.html'>1.7.20</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.20.1/index.html'>1.7.20.1</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.21/index.html'>1.7.21</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.22/index.html'>1.7.22</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.23/index.html'>1.7.23</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.24/index.html'>1.7.24</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.25/index.html'>1.7.25</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.26/index.html'>1.7.26</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.27/index.html'>1.7.27</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.2/index.html'>1.7.2</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.3/index.html'>1.7.3</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.4/index.html'>1.7.4</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.5/index.html'>1.7.5</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.6/index.html'>1.7.6</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.7/index.html'>1.7.7</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.8/index.html'>1.7.8</a></td>
+        <td>1.8 or 1.9</td>
+      </tr>
+      <tr>
+        <td><a href='/files/downloads/1.7.9/index.html'>1.7.9</a></td>
+        <td>1.8 or 1.9</td>
```

Comment:
while its better than previous - its a bit unclear due how 1.8/1.9 were versioned.
maybe it should say 1.8.7/1.9.3 compatible - hopefully starting 1.7.0 it was 1.9.3 (and 1.6.8 was 1.9.2 - not sure)